src/components/ShareModal/index.tsx

```diff
@@ -177,7 +183,13 @@ export default function ShareModal({ url, onShared = () => null }: { url?: strin
           </ButtonWithHoverEffect>
         </Flex>
         <InputWrapper>
-          <input type="text" value={shareUrl} />
+          <input
+            type="text"
+            value={shareUrl}
+            onChange={() => {
```

Comment:
can we remove onchange?

Reply:
Ah yes we can, ESLint is just too whiny, so I added an `empty` func 😂